api/src/main/java/io/opencensus/trace/Tracer.java

```diff
@@ -169,51 +173,50 @@ public final SpanBuilder spanBuilder(String name) {
    *
    * <p>See {@link SpanBuilder} for usage examples.
    *
-   * <p>This <b>must</b> be used to create a {@code Span} when manual Context propagation is used
-   * OR when creating a root {@code Span} with a {@code null} parent.
+   * <p>This <b>must</b> be used to create a {@code Span} when manual Context propagation is used OR
+   * when creating a root {@code Span} with a {@code null} parent.
    *
-   * @param parent The parent of the returned Span. If null the {@code SpanBuilder} will build a
-   *     root {@code Span}.
+   * @param parent The parent of the returned Span. If {@code null} the {@code SpanBuilder} will
+   *     build a root {@code Span}.
    * @param name The name of the returned Span.
    * @return a {@code SpanBuilder} to create and start a new {@code Span}.
-   * @throws NullPointerException if {@code name} is null.
+   * @throws NullPointerException if {@code name} is {@code null}.
```

Comment:
generally, we don't need to add docs like this if using nullable annotation anything not nullable and passed null can be assumed to throw a NPE